std/node/require.ts

```diff
@@ -540,6 +551,32 @@ class Module {
   }
 }
 
+// Polyfills.
+const nativeModulePolyfill = new Map<string, Module>();
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function createNativeModule(id: string, exports: any): Module {
+  const mod = new Module(id);
+  mod.exports = exports;
+  mod.loaded = true;
+  return mod;
+}
+nativeModulePolyfill.set("fs", createNativeModule("fs", nodeFS));
+nativeModulePolyfill.set("util", createNativeModule("util", nodeUtil));
+nativeModulePolyfill.set("path", createNativeModule("path", nodePath));
```

Comment:
Can you add some documentation of this either to std/node/README.md or the jsdoc for makeRequire ?

Reply:
Added. Also renamed `makeRequire` -> `createRequire` and `require.ts` -> `module.ts`. This aligns with https://nodejs.org/dist/latest-v13.x/docs/api/modules.html#modules_module_createrequire_filename better.